crates/superposition/src/organisation/types.rs

```diff
@@ -0,0 +1,55 @@
+use serde::{Deserialize, Serialize};
+use superposition_types::database::models::organisation::OrgStatus;
+use superposition_types::database::models::organisation::Organisation;
+
+// Request payload for creating an organisation
+#[derive(Deserialize)]
+pub struct CreateOrganisationRequest {
+    pub country_code: Option<String>,
+    pub contact_email: Option<String>,
+    pub contact_phone: Option<String>,
+    pub created_by: String,
+    pub admin_email: String,
+    pub status: OrgStatus,
+    pub sector: Option<String>,
+}
+
+// Response type to include `org_id`
+#[derive(Serialize)]
+pub struct CreateOrganisationResponse {
+    pub org_id: String,
+}
+
+// Response struct for single organisation
+#[derive(Serialize)]
+pub struct OrganisationResponse {
+    id: String,
+    country_code: Option<String>,
+    contact_email: Option<String>,
+    contact_phone: Option<String>,
+    admin_email: String,
+    status: OrgStatus,
+    sector: Option<String>,
+    created_at: chrono::NaiveDateTime,
+    updated_at: chrono::NaiveDateTime,
+    created_by: String,
+    updated_by: String,
+}
+
+impl From<Organisation> for OrganisationResponse {
+    fn from(org: Organisation) -> Self {
+        OrganisationResponse {
+            id: org.id,
+            country_code: org.country_code,
+            contact_email: org.contact_email,
+            contact_phone: org.contact_phone,
+            admin_email: org.admin_email,
+            status: org.status,
+            sector: org.sector,
+            created_at: org.created_at,
+            updated_at: org.updated_at,
+            created_by: org.created_by,
+            updated_by: org.updated_by,
+        }
+    }
+}
```

Comment:
why do we need this separately, its same as `Organisation` type, can we please reuse that
type unification was done to avoid these things 😭